src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const object = {};
+
+  for (const i in robot) {
```

Comment:
```suggestion
  for (const key in robot) {
```
i не слишком meaningful